src/gourd/wrapper/mod.rs

```diff
@@ -51,7 +51,7 @@ pub fn wrap(
 
         verify_arch(&program.binary, arch, fs)?;
 
-        let mut cmd = Command::new(&experiment.wrapper);
+        let mut cmd = Command::new(shellexpand::full(&experiment.wrapper)?.to_string());
```

Comment:
Why are we evaluating POSIX shell? Why are we doing this? I really dislike this (a lot)

Reply:
- it causes no errors on other systems 
- I personally want to use `~` in my file paths, plus the error when you use a `~` is misleading 
- I imagine it would let people that know what they're doing do quite a lot with environment variable expansion

I would be willing to remove the variable expansion, but personally I quite enjoy being able to use user home in my paths. if you want to remove that, id want to be convinced it's a problem.

Reply:
1. It doesn't do anything on windows.
2. That's a great argument for removing variable expansion, but not for home dir expansion (~). Do you have any reasoning for removing that one too?

Reply:
2. I do not if you can do it without shell expand or a slimmer dependency.